includes/admin/views/html-admin-page-status-logs-db.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Admin View: Page - Status Database Logs
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+?>
+<form method="get" id="mainform" action="">
+
+	<?php $log_table_list->views(); ?>
+	<?php $log_table_list->search_box( __( 'Search tag', 'woocommerce' ), 'tag' ); ?>
```

Comment:
Is `tag` the correct terminology here? Confused me a moment due to WP tags taxonomy.